src/js/letters/actions/letters.js

```diff
@@ -172,7 +173,10 @@ export function getLetterPdf(letterType, letterName, letterOptions) {
         window.URL.revokeObjectURL(downloadUrl);
         dispatch({ type: GET_LETTER_PDF_SUCCESS, data: letterType });
       },
-      () => dispatch({ type: GET_LETTER_PDF_FAILURE, data: letterType })
+      () => {
+        window.dataLayer.push({ event: 'letter-pdf-failure' });
```

Comment:
Is it possible to also store the specific letter that failed to fetch? In case this it's `letterType`.